File: Home_work/Знакомство с языком Python/Семинар_3/Task_1.py
# Задайте список из нескольких чисел. Напишите программу,
# которая найдёт сумму элементов списка, стоящих на нечётной позиции.
# Пример:
# [2, 3, 5, 9, 3] -> на нечётных позициях элементы 3 и 9, ответ: 12

# Вариант 1
list = [2, 3, 5, 9, 3]
sum = 0
count = 0
for i in list:
    if count % 2 == 1:
        sum += i
    count += 1
print(f'{list} -> {sum}')

# Вариант 2
list = [2, 3, 5, 9, 3]
sum = 0
for i in list[1::2]:
    sum += i
print(f'{list} -> {sum}')

File: Home_work/Знакомство с языком Java/Seminar1/Task1.java
//Вычислить n-ое треугольного число(сумма чисел от 1 до n), n! (произведение чисел от 1 до n)

import java.util.Scanner;

public class Task1 {
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Введите число n: ");
        int n = scanner.nextInt();
        sum(n);
        mult(n);
        scanner.close();
    }

    public static void sum(int n)
    {
        int result = 0;
        for (int i = 1; i <= n; i++)
        {
            result += i;
        }
        System.out.println("Сумма чисел от 1 до " + n + "(включительно) = " + result);
    }

    public static void mult(int n)
    {
        int result = 1;
        for (int i = 1; i <= n; i++)
        {
            result *= i;
        }
        System.out.println("Произведение чисел от 1 до " + n + "(включительно) = " + result);
    }
}